src/Illuminate/Translation/Translator.php

```diff
@@ -217,6 +217,7 @@ protected function makeReplacements($line, array $replace)
         $shouldReplace = [];
 
         foreach ($replace as $key => $value) {
+            $value ??= '';
```

Comment:
I think it should be safely refactored to `(string) $value`.

This allow to use objects that can be transformed into string.

Reply:
It can probably be safely refactored to that but I fail to see how it `allows to use objects that can be transformed into string`. You can already pass a Stringable as the value, as the following test confirms. Am I missing something ?

```php
$t = new Translator($this->getLoader(), 'en');
$t->getLoader()->shouldReceive('load')->once()->with('en', '*', '*')->andReturn([]);
$t->getLoader()->shouldReceive('load')->once()->with('en', 'foo :message baz', '*')->andReturn([]);
$this->assertSame('foo test baz', $t->get('foo :message baz', ['message' => new class {
    public function __toString()
    {
        return 'test';
    }
}]));
```

Reply:
String casting is not the way to go I think/